cypress/Dockerfile

```diff
@@ -1,29 +1,23 @@
-FROM redhat/ubi8:8.9
-RUN dnf -y update
+# ARG YARN_VERSION='1.22.19'
+# ARG NODE_VERSION='20.8.0'
+ARG NODE_VERSION='21.6.1'
```

Comment:
Can we change the version to be `20.8.0` to match the FE?

Reply:
No Cypress need not match frontend of Docusaursus, I have not seen any failtures with latest version, low version will only introduce CVEs